src/storage_types.rs

```diff
@@ -100,6 +101,12 @@ macro_rules! storage_types {
     (@type ($(#[$attr:meta])*) @$M:ident BigRational ($($tt:tt)*)) => {
         storage_type_bigrational!(($(#[$attr])*) @$M ($($tt)*));
     };
+    (@type ($(#[$attr:meta])*) @$M:ident Complex32 ($($tt:tt)*)) => {
+        storage_type_complex32!(($(#[$attr])*) @$M ($($tt)*));
+    };
+    (@type ($(#[$attr:meta])*) @$M:ident Complex64 ($($tt:tt)*)) => {
+        storage_type_complex64!(($(#[$attr])*) @$M ($($tt)*));
+    };
```

Comment:
Move `VV` type aliases to this location.

```suggestion
    (@type ($(#[$attr:meta])*) @$M:ident Complex32 ($($tt:tt)*)) => {
        storage_type_complex32!(($(#[$attr])*) @$M (
            /// Inner storage type.
            #[allow(dead_code)]
            pub type VV = f32;
            $($tt)*));
    };
    (@type ($(#[$attr:meta])*) @$M:ident Complex64 ($($tt:tt)*)) => {
        storage_type_complex64!(($(#[$attr])*) @$M (
            /// Inner storage type.
            #[allow(dead_code)]
            pub type VV = f64;
            $($tt)*));
    };
```